anyscale_provider/hooks/anyscale.py

```diff
@@ -141,24 +141,24 @@ def get_job_status(self, job_id: str) -> JobStatus:
 
         :param job_id: The ID of the job.
         """
-        self.log.info(f"Fetching job status for Job ID: {job_id}")
+        self.log.debug(f"Fetching job status for Job ID: {job_id}")
         return self.client.job.status(id=job_id)
 
     def get_service_status(
         self,
-        service_name: str,
+        name: str,
```

Comment:
can we keep this arg as was and while calling do `service_name=self.name` to avoid a breaking change in the public hook method?